letsencrypt/cli.py

```diff
@@ -44,6 +46,19 @@
 
 logger = logging.getLogger(__name__)
 
+# This is global scope in order to be able to extract type information from
+# it later
+_parser = None
+
+# These are the items which get pulled out of a renewal configuration
+# file's renewalparams and actually used in the client configuration
+# during the renewal process. We have to record their types here because
+# the renewal configuration process loses this information.
+STR_CONFIG_ITEMS = ["config_dir", "log_dir", "work_dir", "user_agent",
```

Comment:
log_dir isn't a thing returned by prepare_and_parse_args, I believe this should be logs_dir